### PR TITLE
docs: update return type of pushrules

### PR DESCRIPTION
**Pull request by JohnVillalovos**
Update the return type of pushrules to surround None with back-ticks
to make it code-formatted.

### CODE REVIEW

File: docs/gl_objects/groups.rst [hunk 1]
```diff
@@ -398,7 +398,7 @@ Create group push rules (at least one rule is necessary)::
 
     group.pushrules.create({'deny_delete_tag': True})
 
-Get group push rules (returns None is there are no push rules)::
+Get group push rules (returns ``None`` is there are no push rules)::
```
Comment (nejch) [inline docs/gl_objects/groups.rst:401]:
```suggestion
Get group push rules (returns ``None`` if there are no push rules)::
```

File: docs/gl_objects/projects.rst [hunk 2]
```diff
@@ -748,7 +748,7 @@ Create project push rules (at least one rule is necessary)::
 
     project.pushrules.create({'deny_delete_tag': True})
 
-Get project push rules (returns None is there are no push rules)::
+Get project push rules (returns ``None`` is there are no push rules)::
```
Comment (nejch) [inline docs/gl_objects/projects.rst:751]:
```suggestion
Get project push rules (returns ``None`` if there are no push rules)::
```

**Review by nejch — CHANGES_REQUESTED** [review]
Thanks @JohnVillalovos just a tiny suggestion while we're making changes here :)

File: docs/gl_objects/groups.rst (hunk 1 above)
Comment (JohnVillalovos) [reply docs/gl_objects/groups.rst:401]: Good catch. Will do.

**Review by nejch — APPROVED** [review]

**Pull request merged**